yt/utilities/lib/bounded_priority_queue.pyx

```diff
@@ -106,21 +219,32 @@ cdef class BoundedPriorityQueue:
                 index = (index - 1) // 2
         return 1
 
+
+def validate_pid():
```

Comment:
We should probably move this function and the one below to the tests.

Reply:
hm, yeah, these are used in the tests but it's easier to have them here as a lot of the queue functions are cdefs so not callable from python. I call these and compare the results to some answers in the tests

Reply:
Ah, I see. Can you add a comment to that effect? Another option would be to add a cython file to the tests themselves, and export the cdef functions you need in a `.pxd` file.

Reply:
I think adding the comment is probably easier and more consistent with the current format, but you know the code better than me

Reply:
The comment is fine :)